js/index.js

```diff
@@ -40,3 +40,71 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//Selectors
+let cta =  document.getElementById("cta-img");
+let nav = document.querySelectorAll("a");
+let h1 = document.querySelector("h1");
+let button = document.querySelector("button");
+let mcimage = document.getElementById("middle-img")
+let h4 = document.querySelectorAll("h4");
+let txtcontent = document.querySelectorAll("p");
+
+
+
+
+
+//Updates
+cta.setAttribute('src', siteContent["cta"]["img-src"])
+mcimage.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+let navArray = Array.from(nav); 
+navArray[0].textContent = siteContent["nav"]["nav-item-1"]; 
+navArray[1].textContent = siteContent["nav"]["nav-item-2"]; 
+navArray[2].textContent = siteContent["nav"]["nav-item-3"]; 
+navArray[3].textContent = siteContent["nav"]["nav-item-4"]; 
+navArray[4].textContent = siteContent["nav"]["nav-item-5"]; 
+navArray[5].textContent = siteContent["nav"]["nav-item-6"]; 
+
+h1.textContent = siteContent["cta"]["h1"];
+h1.style.width = "41%";
```

Comment:
Nice job styling responsively. I know it isnt part of the assignment but always nice to see.